zerver/openapi/zulip.yaml

```diff
@@ -11518,6 +11518,24 @@ paths:
                               type: string
                               description: |
                                 The name of the bot.
+                            display_name:
```

Comment:
Is `null` possible here?

Reply:
No, it cannot be `null`. 

The following lines of code in the `__init__` function of the `Integration` class which is the parent class of `WebhookIntegration` make sure that each integration has a `display_name`:

https://github.com/zulip/zulip/blob/d205850d54f73d648488bc1dbd37cca2dfdef272/zerver/lib/integrations.py#L104-L106